examples/arm/raspberry_pi/pico2/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I do not remember what we agreed on for the home for this? Given we might also enable this for RISCv (not just arm)